common/djangoapps/dark_lang/middleware.py

```diff
@@ -102,13 +113,17 @@ def _clean_accept_headers(self, request):
         if accept is None or accept == '*':
             return
 
-        new_accept = ", ".join(
-            self._format_accept_value(lang, priority)
-            for lang, priority
-            in dark_parse_accept_lang_header(accept)
-            if self._is_released(lang)
-        )
+        print "Getting new acceptance headers"
+
+        new_accept = []
+        for lang, priority in dark_parse_accept_lang_header(accept):
+            fuzzy_code = self._fuzzy_match(lang.lower())
```

Comment:
lang is lowered here, but I didn't see where the released_langs are lowered, maybe I just missed it?